vocab/credentials/v2/issuer-dependent.html

```diff
@@ -66,15 +68,57 @@
     <section>
         <h2>Issuer Dependent Terms</h2>
         <p>
-            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> of a Verifiable Credential, as defined in [vc-data-model], provides a mapping from the standard terms used in credentials to the vocabulary items defined by the Verifiable Credentials Working Group through different vocabularies, such as the <a href="https://www.w3.org/2018/credentials/">Verifiable Credentials Vocabulary</a>. Applications using additional, user-dependent vocabularies of their own are encouraged to create their own formal vocabularies, and corresponding JSON-LD context definitions, to ensure the interoperability of the credentials they produce and/or consume.
+            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> of a Verifiable Credential, 
+            as defined in [vc-data-model], provides a mapping from the standard terms used in credentials to the vocabulary 
+            items defined by the Verifiable Credentials Working Group through different vocabularies, 
+            such as the <a href="https://www.w3.org/2018/credentials/">Verifiable Credentials Vocabulary</a>. 
+            Applications using additional, issuer-dependent vocabularies of their own are encouraged to 
+            create their own formal vocabularies, and corresponding JSON-LD context definitions, 
+            to ensure the interoperability of the credentials they produce and/or consume.
```

Comment:
We should highlight an important reason for using issuer-independent vocabularies and contexts here:

```suggestion
            to ensure the interoperability of the credentials they produce and/or consume. Using
            issuer-independent vocabularies reduces the burden on consumers of verifiable
            credentials, who do not need to update software when different issuers reuse the
            same shared vocabularies.
```

Reply:
I agree with the intention, but not this specific text.

It seems if would be better to no mention "updating software" here.

Perhaps a citation on the using widely adopted vocabularies such as schema.org would be better.

Reply:
I think it's important to highlight the burden on consumers who will have to make adjustments for every different issuer when shared (issuer-independent) vocabularies are not used. If the above language doesn't work for you -- could you suggest something else that communicates this?

Reply:
> I think it's important to highlight the burden on consumers who will have to make adjustments for every different issuer when shared (issuer-independent) vocabularies are not used. 

I don't think I understand what you are saying here... I can imagine `consumers` never seeing an IRI for any of the terms the core data model uses, since the base media type is JSON-LD, and the IRIs only show up to consumers of RDF.

> If the above language doesn't work for you -- could you suggest something else that communicates this?

I can try:

If you plan on consuming Verifiable Credentials as RDF, for example, after converting from `application/vc+ld+json` to `application/n-quads`, please be aware that the core data model is JSON-LD and that the issuer may not have intended for `application/n-quads` to be consumed when they added integrity protection to their Verifiable Credential.

This can cause the predicates and types that you process to be potentially ambiguous, in cases where multiple issuers used the same terms in JSON-LD, without defining them, in `issuer independent` contexts.

Reply:
@OR13,

> I don't think I understand what you are saying here... I can imagine consumers never seeing an IRI for any of the terms the core data model uses, since the base media type is JSON-LD, and the IRIs only show up to consumers of RDF.

This has little to do with IRIs and much more to do with "context". If a VC does not use its own `@context`, then the only "context" available to consumers is who the issuer is. That means the terms, whatever they may be, are issuer-dependent.

It means consumers must understand N "issuers + their credential types". This is a much higher burden than understanding N credential types (via `@context` values), regardless of the issuer.

When you write digital wallet software that is supposed to help users make decisions ... it must check a well-known list of issuers and their VC types to determine what the meaning of a VC is. This is a different class of problem than having to check a well-known list of JSON-LD contexts. Every new JSON-LD context that a wallet recognizes onboards any issuer that uses it. Every VC that uses issuer-independent terms requires the wallet to understand who that issuer is and what they meant by that specific type of credential. Additionally, they must track down the documentation for understanding the terms in issuer-priorietary ways.

The latter case increases the burden considerably for digital wallet providers. One can only expect them to put in the effort for the largest VC issuers -- which will lead to market concentration problems. People who care about this should be aware of this -- so that they might help avoid it. Of course, market players that want to do it on purpose will not be dissuaded, but they are not the target audience of the language here.

So, this has nothing to do with "if you plan on using VCs as RDF" or "n-quads", etc. This has to do with enabling software to determine types based on JSON-LD `@context` -- including writing software that uses VCs as JSON.

Reply:
> Every VC that uses issuer-independent terms requires the wallet to understand who that issuer is

You meant `issuer-dependent` here, I think?

Reply:
@TallTed, yes -- thanks I've applied the edit.